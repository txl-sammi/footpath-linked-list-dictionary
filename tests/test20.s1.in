Palmerston Street between Rathdowne Street and Drummond Street
Morrah Street between Fitzgibbon Street and Wimble Street
Lygon Street between Argyle Place South and Pelham Street
Canning Street between Palmerston Street and Pitt Street
Gatehouse Street between Morrah Street and Bayles Street
Franklin Street between Elizabeth Street and Queen Street
Queensberry Street between Capel Street and Howard Street
Lygon Street between Victoria Street and Queensberry Street
Drummond Street between Faraday Street and Elgin Street
David Street between Elgin Street and Palmerston Street
Carlton Street between Nicholson Street and Canning Street
