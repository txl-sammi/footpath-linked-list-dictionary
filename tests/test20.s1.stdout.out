Palmerston Street between Rathdowne Street and Drummond Street --> 1
Morrah Street between Fitzgibbon Street and Wimble Street --> 1
Lygon Street between Argyle Place South and Pelham Street --> 1
Canning Street between Palmerston Street and Pitt Street --> 1
Gatehouse Street between Morrah Street and Bayles Street --> 1
Franklin Street between Elizabeth Street and Queen Street --> 1
Queensberry Street between Capel Street and Howard Street --> 1
Lygon Street between Victoria Street and Queensberry Street --> 1
Drummond Street between Faraday Street and Elgin Street --> 1
David Street between Elgin Street and Palmerston Street --> 1
Carlton Street between Nicholson Street and Canning Street --> 1
